prisma/seed.ts

```diff
@@ -30,20 +48,40 @@ const clearConnections = async () => {
               .fill(undefined)
               .map((_, idx) => ({ id: `${idx}` })),
           },
+          connectedTo: {
+            disconnect: new Array(70)
+              .fill(undefined)
+              .map((_, idx) => ({ id: `${idx}` })),
+          },
         },
-      });
-    })
+      })
+    )
   );
 };
 
 /**
  * Generates favorites for users in our database.
  */
-const generateFavorites = async () => {
+const generateConnectedFields = async () => {
```

Comment:
nit but can we move the functionality for favorites and invites into seperate helper functions?